backend/models.py

```diff
@@ -10,13 +10,15 @@ class BedAssignmentResponse(BaseModel):
     patient_id: int
     patient_name: str
     sickness: str
+    PESEL: str
     days_of_stay: int
 
 
 class PatientQueueResponse(BaseModel):
     place_in_queue: int
     patient_id: int
     patient_name: str
+    PESEL: str
```

Comment:
Keep consistent naming convection 
```suggestion
    pesel: str
```